components/NewsItem.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react'
 import Upvote from './Upvote'
+import Points from './Points'
 
 class NewsItem extends React.Component {
   constructor (props, context) {
     super(props, context)
     this.state = {upvoted: this.props.upvoted}
+    this.state = {count: this.props.count}
   }
 
   handleClick () {
     this.setState({
-      upvoted: !this.state.upvoted
+      upvoted: !this.state.upvoted,
+      count: this.state.count + 1
```

Comment:
This is going to increment on every click, so if you click on, then off, then on again, it'll have incremented three times.